CredScanSuppressions.json

```diff
@@ -2,15 +2,15 @@
 	"tool": "Credential Scanner",
 	"suppressions": [
 		{
-			"file": "server/routerlicious/node_modules/.pnpm/secure-keys@1.0.0/node_modules/secure-keys/test/test.secret.key",
+			"file": "node_modules/.pnpm/secure-keys@1.0.0/node_modules/secure-keys/test/test.secret.key",
```

Comment:
Do these three files appear in both the old and the new path? What is the file path relative to?

Reply:
The full paths are technically `server/routerlicious/<bla>`, `server/historian/<bla>`, and `server/gitrest/<bla>`, but the way CredScan uses the suppressions file we just need to specify a postfix, which is nice for our mono-repo setup so we don't have to ignore these separately in every place they show up as a transitive dependency. The full paths are relative to the root of the repo (which I found a bit confusing, because in the previous PR I had to copy the `CredScanSuppressions.json` file from the root of the repo to the build directory for each pipeline, i.e. `server/routerlicious`, `server/historian`, `server/gitrest`, but somehow the tool still says "I don't like this file <full path from the root of the repo>").